Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/question.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
```

Comment:
Weird comment but I would stray away from using .jsx and just use .js. Ive run into some issues with various dependencies before because they only look for .js files. There are usually easy work arounds, but .jsx doesnt really offer anything to make it worth the potential hassle imo

Reply:
Sounds good will switch!